docs/get-started/install.md

```diff
@@ -27,7 +27,9 @@ title: 'Install Storybook'
 
 <summary><code>sb init</code> is not made for empty projects</summary>
 
-Storybook needs to be installed into a project that is already setup with a framework. It will not work on an empty project. There are many ways to bootstrap an app in a given framework including:
+Storybook needs to be installed into a project that is already setup with a framework. It will not work on an empty project. Make sure you are in the root directory of your project as well. For example: if you are using angular, run this command in the the directory that your angular.json is. 
+
+There are many ways to bootstrap an app in a given framework including:
```

Comment:
@safwaan-menk, instead of adding your suggestion here, I think we could go with the following:
- Leave it as it is initially to prevent "tilting the scales" (pardon the bad pun) for the documentation, leaving it as agnostic as much as possible, and adjust the following [file](https://github.com/storybookjs/storybook/blob/next/docs/get-started/installation-problems/angular.mdx) to include your proposal, possibly turning it into:
```md
- You can also setup Storybook manually through the Storybook CLI. Add the `--type angular` flag when you initialize Storybook in your project.
- If there's an installation problem, check the [README for the Angular framework](../../app/angular/README.md).
- If you're having issues installing Storybook, check if you're running the install command in the root directory of your project, or if you're working with a monorepo from the directory where the angular.json is located.
```